### DISCUSSION TITLE
建议接入Bing壁纸API

### DISCUSSION THREAD

**Discussion by xaoxuu** [volantis-x/hexo-theme-volantis#524] [Q&A] [answered]
## 需求描述

可以用作首页封面

API: https://github.com/xCss/bing

> 有其它更好的 API 也可以

**Comment by heson525** [comment]
还是喜欢扁平风，对这种拟实的风景不太感冒。😁

**Comment by MHuiG** [comment]
https://uploadbeta.com/api/pictures/random/?key=BingEverydayWallpaperPicture

<blockquote></blockquote>

**Comment by MHuiG** [comment]
https://api.ixiaowai.cn/api/api.php

<blockquote></blockquote>

**Comment by MHuiG** [comment]
https://api.ixiaowai.cn/mcapi/mcapi.php

<blockquote></blockquote>

**Comment by MHuiG** [comment]
https://api.ixiaowai.cn/gqapi/gqapi.php

<blockquote></blockquote>

**Comment by MHuiG** [comment]
https://source.unsplash.com/1600x900/?nature,water

<blockquote></blockquote>

**Comment by MHuiG** [comment]
自定义随机API 可？

**Comment by MHuiG** [comment, marked as answer]
```
cover:
  background: https://uploadbeta.com/api/pictures/random/?key=BingEverydayWallpaperPicture
```

好像这样就搞定了

[a comment by an automated account is not shown]

**Comment by xaoxuu** [comment]
我觉得可以注释到配置文件里
